index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>PORTFOLIO</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  </head>
+  <body>
+    <header class="hero">
+      <nav>
+        <a href="#" class="logo">Welcome</a>
+        <ul class = "nav-list">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <label for="check" class="hambuger-btn">
+          <i class = "fas fa-bars"></i>
+        </label>
+      </nav>
+    </header>
+    <section>
+      <div class="content">
+        <h1>Hey there. I'm Bobson</h1>
+        <h2>I'm a software developer</h2>
+        <P class = "supporting-text">I can help you build a feature, product or a website.
+        Look through some of my work and experience!
+        If you like what you see and have a project you need coded, dont
+          hesitate to contact me.
+        </p>
+      </div>
+      <section>
+      <ul class="socialmedia">
+        <li>
+          <a href="https://twitter.com/kalungibobsonj">
+              <img src="./images/socialmedia icons/twitter.png" alt="twitter icon">
+          </a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/kalungi-j-bobson/">
+              <img src="./images/socialmedia icons/linkedin.png" alt="linkedin icon">
+          </a>
+        </li>
+        <li>
+          <a href="https://medium.com/@kalungijbobson">
+              <img src="./images/socialmedia icons/mail.png" alt="Medium icon" />
+          </a>
+      </li>
+      <li>
+      <a href="https://github.com/kalungi-j-bobson">
+        <img src="./images/socialmedia icons/github.png" alt="github icon">
+    </a>
+   </li>
+   <a href="#">
+    <img src="./images/socialmedia icons/angelist 1.png" alt="angelist icon">
+</a>
+</li>
+    </ul>
+  </section>
+      <section class="disabled">
+        <img
+          src="./images/Disabled.png"
+          alt="disabled vector"
+        />
+      </section>
```

Comment:
- To improve your HTML semantics, it is always important that all opening tags have a corresponding closing tag, Please include a closing tag for the `body` tag 😄 
- According to the project requirements, the mobile version should be for all screen less than `768px`, please make sure that your mobile site fits for all screens less than `768px`. 

Your project:
<img width="653" alt="hrhzrh" src="https://user-images.githubusercontent.com/73607512/200133689-7d9ea5fa-225e-46bc-acda-20c211eef586.PNG">